src/main/java/edu/greenblitz/tobyDetermined/Nodesssss/NodeBase.java

```diff
@@ -0,0 +1,115 @@
+
+
+            nodeMap.put(GRIPER_OPEN, new GriperNode());
+            nodeMap.put(GRIPER_CLOSE, new GriperNode());
+            listSystemsPos.add(GRIPER_OPEN);
+            listSystemsPos.add(GRIPER_CLOSE);
+
+            nodeMap.get(GRIPER_OPEN).addNeighbors(new SystemsPos[]{GRIPER_CLOSE});
+            nodeMap.get(GRIPER_CLOSE).addNeighbors(new SystemsPos[]{GRIPER_OPEN});
+
+
+            nodeMap.get(GRIPER_OPEN).setOtherSystemMustBeToEnter2(new SystemsPos[]{ARM_ZG, ARM_MID, ARM_HIGH});
+            nodeMap.get(GRIPER_OPEN).setOtherSystemMustBeToOut2(new SystemsPos[]{ARM_ZG, ARM_MID, ARM_HIGH});
+            nodeMap.get(GRIPER_CLOSE).setOtherSystemMustBeToEnter2(new SystemsPos[]{ARM_ZG, ARM_MID, ARM_HIGH});
+            nodeMap.get(GRIPER_CLOSE).setOtherSystemMustBeToOut2(new SystemsPos[]{ARM_ZG, ARM_MID, ARM_HIGH});
+
+            nodeMap.get(CLIMBING_ROMY).setOtherSystemMustBeToEnter2(new SystemsPos[]{ARM_ZG});
+            nodeMap.get(CLIMBING_NOAM).setOtherSystemMustBeToEnter2(new SystemsPos[]{ARM_ZG});
+            nodeMap.get(CLIMBING_ROMY).setOtherSystemMustBeToEnter3(new SystemsPos[]{GRIPER_OPEN});
+            nodeMap.get(CLIMBING_NOAM).setOtherSystemMustBeToEnter3(new SystemsPos[]{GRIPER_OPEN});
+
+            nodeMap.get(ARM_LOW).setOtherSystemMustBeToEnter2(new SystemsPos[]{GRIPER_OPEN});
+            nodeMap.get(ARM_LOW).setOtherSystemMustBeToOut2(new SystemsPos[]{GRIPER_OPEN});
+            nodeMap.get(ARM_LOW).setOtherSystemMustBeToEnter3(new SystemsPos[]{CLIMBING_ROMY});
+            nodeMap.get(ARM_LOW).setOtherSystemMustBeToOut3(new SystemsPos[]{CLIMBING_ROMY});
+
+
+        }
+    }
+
+    public static class SetCosts {
+        static LinkedList<Pair<String, Double>> costList = new LinkedList<>();
+
+        static {
+            costList.add(new Pair<>(ARM_LOW + "-" + ARM_MID, 3.0));
+            costList.add(new Pair<>(ARM_LOW + "-" + ARM_HIGH, 2.0));
+            costList.add(new Pair<>(ARM_LOW + "-" + ARM_ZG, 7.0));
+            costList.add(new Pair<>(ARM_HIGH + "-" + ARM_ZG, 1.0));
+            costList.add(new Pair<>(ARM_HIGH + "-" + ARM_MID, 10.0));
+            costList.add(new Pair<>(ARM_ZG + "-" + ARM_MID, 12.0));
+            costList.add(new Pair<>(GRIPER_CLOSE + "-" + GRIPER_OPEN, 15.0));
+            costList.add(new Pair<>(CLIMBING_NOAM + "-" + CLIMBING_ROMY, 17.0));
+        }
+    }
+
+    public static class Constants {
+        static public String systemName1 = "ARM";
+        static public String systemName2 = "GRIPER";
+        static public String systemName3 = "CLIMBING";
+        static public SystemsPos system1StartingNode = ARM_ZG;
+        static public SystemsPos system2StartingNode = GRIPER_CLOSE;
+        static public SystemsPos system3StartingNode = CLIMBING_ROMY;
+    }
+
+    public static class important {
+        NodeArm a = new NodeArm(0, 'r' + 'o' + 'm' + 'y' + ' ' + 'i' + 's' + ' ' + 'a' + 'u' + 't' + 'i' + 's' + 't' + 'i' + 'c');
+    }
```

Comment:
Used for debug?

Reply:
yes this is very important this is why its in important, to make it clear by informative name 💩.